base.html

```diff
@@ -12,10 +12,21 @@
       <!-- display -->
       <input type="text" class="display" disabled />
       <!-- Button to iterate through turns -->
-      <button class="button" onclick="turn()">Next Turn</button>
+      <button class="button" onclick="advanceTurn()">Next Turn</button>
     </div>
 
-    <div id="cont"></div>
+    <table id="table" class="fixed_headers">
```

Comment:
I suggest loading your CSS and JS using `<link rel="stylesheet" href="style.css" type="text/css">` and `<script src="myscripts.js"></script>`